test/test.spec.js

```diff
@@ -70,16 +71,12 @@ describe('The CLI', function () {
 });
 
 describe('The server', function () {
-  var server;
-  before('Start running the server', function() {
+  let server;
+  before('Start running the server', function () {
```

Comment:
Ahh, interesting. Jest doesn't really use `this` within tests so I haven't encountered issues using arrow functions in my test at work  :)